### PR TITLE
Increase retries and backoff of loadbalancer service smoke test

### PR DESCRIPTION
**Pull request by pliurh**
**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

**Comment by pmtk** [comment]
FYI There's also load balancer test in Robot Framework: https://github.com/openshift/microshift/blob/main/test/suites/load-balancer.robot
I think that is the one that is flaky

File: e2e/tests/0031-loadbalancer-smoke-test.sh [hunk 1]
```diff
@@ -25,6 +25,8 @@ oc create -f "${SCRIPT_PATH}/assets/microshift-test.yaml"
 oc create service loadbalancer microshift-test --tcp=5678:8080
 oc wait pods -l app=microshift-test --for condition=Ready --timeout=60s
 
+timeout 10s bash -c "until kubectl get service microshift-test --output=jsonpath='{.status.loadBalancer}' | grep ingress; do sleep ${BACKOFF} ; done"
+
 for _ in $(seq "${RETRIES}"); do
```
Comment (ggiguash) [inline e2e/tests/0031-loadbalancer-smoke-test.sh:30]: Should we just increase the RETRIES=10 and BACKOFF=6, so that we test the service for 1m.

**Pull request merged**